src/Company/Serializers/CompanyNormalizer.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Company\Serializers;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use VSV\GVQ_API\Company\Models\Company;
+use VSV\GVQ_API\Company\Models\TranslatedAlias;
+
+class CompanyNormalizer implements NormalizerInterface
+{
+    /**
+     * @var TranslatedAliasNormalizer
+     */
+    private $translatedAliasNormalizer;
+
+    /**
+     * @param TranslatedAliasNormalizer $translatedAliasNormalizer
+     */
+    public function __construct(
+        TranslatedAliasNormalizer $translatedAliasNormalizer
+    ) {
+        $this->translatedAliasNormalizer = $translatedAliasNormalizer;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function normalize($company, $format = null, array $context = array()): array
+    {
+        /** @var Company $company */
```

Comment:
Ik denk dat we de typehinting in de PHPdoc boven de methode kunnen zetten `@param TranslatedAlias $translatedAlias`. Je mag dit ook consistent maken bij de reeds bestaande normalizers.